README.md

```diff
@@ -5,24 +5,26 @@ This project is the frontend of Gordon 360 in React. [The retired frontend](http
 ## Contents
 
 - [Getting Started](#getting-started)
-- [Editor Recommendations](#editor-recommendations)
-- [Libraries](#libraries)
-- [Connect Local Backend to React](#connect-local-backend-to-react)
-- [Server Notes](#server-notes)
+  - [Front End Development](#front-end)
+  - [Back End Development](#connect-local-backend-to-react)
+  - [Server Notes](#server-notes)
+  - [Editor Recommendations](#editor-recommendations)
+  - [Libraries](#libraries)
 
 - [Code Style](#code-style)
 - [Dependencies](#dependencies)
 - [Project File Organization](#file-organization)
-- [Enviroment Variables](#environment-variables)
+- [Environment Variables](#environment-variables)
 - [Testing](#testing)
 - [Deployment](#deployment)
+  - [Contributing](#contributing)
 - [Known Issues](#known-issues)
 
 ## Getting Started
 
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app). Read the user guide [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
 
-Make sure Node.js is set up on your machine. Travis-CI uses version `9.11.1`; later versions may work as well.
+Make sure Node.js is set up on your machine. Travis-CI uses version `9.11.1`; later versions may work as well. The following procedures will install nvm, a node.js version manager onto your machine. An npm installation may also work, but nvm will allow installation of this particular version to be used.
```

Comment:
I think it would be clearer to change "but nvm ... used." to say "but nvm will allow easier installation and use of this particular version."